BGL/include/CGAL/draw_face_graph.h

```diff
@@ -0,0 +1,183 @@
+  {
+    if (fh==boost::graph_traits<Graph>::null_face()) // use to get the mono color
+      return CGAL::Color(100, 125, 200); // R G B between 0-255
+
+    return get_random_color(CGAL::get_default_random());
+  }
+};
+
+class SimpleFaceGraphViewerQt : public Basic_viewer_qt
+{
+  using Base = Basic_viewer_qt;
+
+public:
+  SimpleFaceGraphViewerQt(QWidget* parent) :
+    Base(parent, "", false, true, true, true, false),
+    m_compute_elements_impl([]{})
+  {
+  }
+
+  /// Construct the viewer.
+  /// @param amesh the surface mesh to view
+  /// @param title the title of the window
+  /// @param anofaces if true, do not draw faces (faces are not computed; this can be
+  ///        usefull for very big object where this time could be long)
+  template <typename SM>
+  SimpleFaceGraphViewerQt(QWidget* parent,
+                          const SM& amesh,
+                          const char* title="Basic Surface_mesh Viewer",
+                          bool anofaces=false) :
+    SimpleFaceGraphViewerQt(parent, amesh, title, anofaces, DefaultColorFunctorFaceGraph())
+  {
+  }
+
+  template <typename SM, typename ColorFunctor>
+  SimpleFaceGraphViewerQt(QWidget* parent,
+                          const SM& amesh,
+                          const char* title,
+                          bool anofaces,
+                          ColorFunctor fcolor) :
+    // First draw: no vertex; edges, faces; mono-color; inverse normal
+    Base(parent, title, false, true, true, true, false),
+    m_compute_elements_impl(compute_elements_functor(amesh, anofaces, fcolor))
+  {
+  }
+
+  void init() override {
+    compute_elements();
+    Base::init();
+  }
+
+  void compute_elements() {
+    m_compute_elements_impl();
+  }
+
+  template <typename SM, typename ColorFunctor>
+  void set_face_graph(const SM& amesh,
+                      bool anofaces,
+                      ColorFunctor fcolor) {
+    m_compute_elements_impl = compute_elements_functor(amesh, anofaces, fcolor);
+  }
+
+  template <typename SM, typename ColorFunctor>
+  void set_face_graph(const SM& amesh,
+                      bool anofaces=false) {
+    set_mesh(amesh, anofaces, DefaultColorFunctorFaceGraph());
+  }
+protected:
+  template <typename SM, typename ColorFunctor>
+  std::function<void()>
+  compute_elements_functor(const SM& sm,
+                           bool anofaces,
+                           ColorFunctor fcolor)
+  {
+    using Point =
+        typename boost::property_map_value<SM, CGAL::vertex_point_t>::type;
+    using Kernel = typename CGAL::Kernel_traits<Point>::Kernel;
+    using Vector = typename Kernel::Vector_3;
+
+    auto vnormals = get(CGAL::dynamic_vertex_property_t<Vector>(), sm);
+    {
+      // temporary face property map needed by `compute_normals`
+      auto fpm = get(CGAL::dynamic_face_property_t<Vector>(), sm);
+
+      CGAL::Polygon_mesh_processing::compute_normals(sm, vnormals, fpm);
+    }
```

Comment:
As a reply to https://github.com/CGAL/cgal/pull/4876#discussion_r461545678:
@sloriot This is where I use `<CGAL/Dynamic_property_map.h>` and `<CGAL/Polygon_mesh_processing/compute_normal.h>`.

Reply:
Which are already included in this header...